.gitignore

```diff
@@ -90,4 +90,4 @@ default.profraw
 .husky/_/
 
 # Lighthouse CI
-.lighthouseci/
```

Comment:
Sorry for that, it may be because I was using gitpod to develop the changes and added `.gitpod.yml` but after the changes, I thought of removing it and I forgot to remove that blank line. Anyways, thanks for the compliment 🙌